README.md

```diff
@@ -93,13 +93,22 @@ Wechaty already held lots of talk and got a lot of blogs in the past 4 years, he
 
 ```javascript
 
-import { WechatyBuilder } from 'wechaty'
+import { WechatyBuilder } from "wechaty";
+
+const wechatyBot = WechatyBuilder.build({}); // get a Wechaty instance
+
+wechatyBot
+  .on("scan", (qrcode, status) =>
```

Comment:
Could you please put all lines in the `scan` listener function in one line?

After that, I think this PR is good to go as a workaround for now.